src/main/java/nl/tudelft/contextproject/logging/Log.java

```diff
@@ -15,32 +15,17 @@
  * Class for conveniently logging messages.
  */
 public class Log {
-	//The default format for files: "HH:MM:SS [LEVEL]: Message", optionally followed by an exception and it's stacktrace.
+	//The default format for files: "HH:MM:SS [LEVEL]: Message", optionally followed by an exception and its stacktrace.
 	public static final String FORMAT_FILE = "%1$tH:%1$tM:%1$tS [%3$s]: %4$s%5$s%n";
-
-	//Log level for fatal messages
 	public static final Level FATAL = createLogLevel("FATAL", 2000);
-	
-	//Global set of loggers, used to properly close FileHandlers.
 	private static final Map<String, Log> LOGGERS = new ConcurrentHashMap<>();
-	
-	//Handler for logging messages to the console.
 	private static final ConsoleHandler CONSOLE_HANDLER;
 	
 	static {
-		//Create the console handler, set the formatter, and add it to the logger
 		CONSOLE_HANDLER = new ConsoleHandler();
-		
-		//Log messages to the console like:
-		//14:59:26 [WARNING] [ENGINE]: There is a problem!
-		//Exception - Stacktrace
 		LogFormatter formatter = new LogFormatter("%1$tH:%1$tM:%1$tS [%3$s] [%2$s]: %4$s%5$s%n");
```

Comment:
The comments above this provide useful information about the format and should be re-added.